server/src/main/java/org/cloudfoundry/identity/uaa/provider/saml/SamlAuthenticationFilterConfig.java

```diff
@@ -225,8 +226,9 @@ Saml2LogoutResponseFilter saml2LogoutResponseFilter(RelyingPartyRegistrationReso
     @Bean
     Saml2LogoutRequestFilter saml2LogoutRequestFilter(RelyingPartyRegistrationRepository relyingPartyRegistrationRepository,
                                                       UaaAuthenticationFailureHandler authenticationFailureHandler,
-                                                      CookieBasedCsrfTokenRepository loginCookieCsrfRepository) {
-        RelyingPartyRegistrationResolver relyingPartyRegistrationResolver = new DefaultRelyingPartyRegistrationResolver(relyingPartyRegistrationRepository);
```

Comment:
should be able to use DefaultRelyingPartyRegistrationResolver instead of creating a new one